Assets/__Scripts/UI/PersistentUI.cs

```diff
@@ -386,42 +391,90 @@ public enum DialogBoxPresetType
     [Serializable]
     public class MessageDisplayer {
 
+        public class NotificationMessage
+        {
+            private AsyncOperationHandle<string>? _localisable;
+
+            public float waitTime = 2.0f;
+            public string message;
+            public bool cancelled = false;
+            public bool skipFade = false;
+            public readonly DisplayMessageType type;
```

Comment:
This isn't going to be a PR killer, but I will say that sometime after Zenject, I will be more strict with the [.NET Naming Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines)

(TL;DR: please use `PascalCase` with public members, kthx)